website/docs/d/is_volumes.html.markdown

```diff
@@ -0,0 +1,154 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_volumes"
+description: |-
+  Get information about VolumeCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_volumes
+
+Provides a read-only data source for VolumeCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_volumes" "example" {
+}
+```
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the VolumeCollection.
```

Comment:
same as other doc, replace * with - and remove required/optional from exported attributes